packs/mmonit/actions/list_events.yaml

```diff
@@ -0,0 +1,63 @@
+---
+name: list_events
+runner_type: run-python
+description: Returns a list of events stored in M/Monit.
+enabled: true
+entry_point: list_events.py
+parameters:
+  active:
+    type: integer
+    description: 'The active events filter. Possible values: 0=show all events, 1=show only active errors, 2=show only active errors but exclude user dismissed errors'
+    required: false
+  hostid:
+    type: integer
+    description: 'Show only events from hosts with matching host id.'
+    required: false
+  hostgroupid:
+    type: integer
+    description: 'Show only events for hosts that are member of the given hostgroupid.'
+    required: false
+  servicenameid:
+    type: integer
+    description: 'Show only events for the given service name id.'
+    required: false
+  servicegroupid:
+    type: integer
+    description: 'Show only events for the given servicegroup id.'
+    required: false
+  servicetype:
+    type: integer
+    description: 'Show only events for the given service type. Possible values: 0=filesystem, 1=directory, 2=file, 3=process, 4=remote host, 5=system, 6=fifo, 7=program.'
```

Comment:
If there is a small set of possible values which doesn't changes, there is an "enum" attribute you can use.

See the following link for an example - https://github.com/StackStorm/st2contrib/blob/master/packs/jira/actions/create_issue.yaml#L12